docs/screencasts/generate-readme-svg.sh

```diff
@@ -19,9 +19,10 @@ docker run -it \
 
 # Fix meta data: width and height are always zero in Docker produced cast files.
 # Header is the first lint of cast file in JSON format, we read, fix and write it.
-head recordings/github-readme.cast -n 1 | yq e -M '.width = 95 | .height = 17' - > new_header.cast
+head -n 1 recordings/github-readme.cast | yq -M '.width = 95 | .height = 17 | . + {"idle_time_limit": 0.5}' - > new_header.cast
 # Then append everything, expect first line from original cast file.
 tail -n+2 recordings/github-readme.cast >> new_header.cast
+# Check if header containers "idle_time_limit": 0.5
```

Comment:
misplaced comment?

Reply:
Yeah, you can check the header after either step doesn't really matter.
Ideally, we wouldn't even need that comment.
But I gave up to make `yq` insert `idle_time_limit` correctly. It always omits the double quotes around the object key...